api/src/main/java/org/pmiops/workbench/db/model/Participant.java

```diff
@@ -0,0 +1,71 @@
+package org.pmiops.workbench.db.model;
+
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.pmiops.workbench.model.CohortStatus;
+
+import javax.persistence.Column;
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name = "participant_cohort_status")
+public class Participant {
```

Comment:
Can we rename to this ParticipantCohortStatus?

"Participant" seems like it should be reserved more general information about a participant.